query-engine/query-engine-wasm/src/wasm/engine.rs

```diff
@@ -180,7 +179,8 @@ impl QueryEngine {
             .validate_that_one_datasource_is_provided()
             .map_err(|errors| ApiError::conversion(errors, schema.db.source()))?;
 
-        let enable_tracing = config.preview_features().contains(PreviewFeature::Tracing);
+        // Telemetry panics on timings if preview feature is enabled
+        let enable_tracing = false; // config.preview_features().contains(PreviewFeature::Tracing);
```

Comment:
Ok, and here you are disabling tracing, so in spite of the feature being enabled, things don't explode.